CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # OMC3 Changelog
 
+#### 2024-07-08 - v0.15.0 - _jdilly_
+
+- PINNING NUMPY TO < 2.0.0
+
+- Changed:
+  - Model creation:
+    - removed hard-coded `knobs.madx` from `lhc`
+    - removed `corrections.madx` from `lhc` best-knowledge model
```

Comment:
let's say that it was unused

Reply:
It was not unused by the code. We just never used it, but we could have. But it comes from a time when you couldn't manually specify the modifiers and also there was only one... So now this is all way more flexible